app/posts/register-trainee-teachers/2020-12-15-recording-training-outcomes-iteration-2.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Recording training outcomes (iteration 2)
+description: Removing the option for failure when recording training outcomes
+date: 2020-12-15
+---
+
+In the [previous iteration of recording training outcomes](/register-trainee-teachers/recording-training-outcomes/) we made the assumption that  our users sometimes withdraw trainees when recording that they did not meet the QTS standards.
```

Comment:
I feel like this should be reframed - we thought we needed to provide a way of recording outcomes other than pass.  I don't think withdrawal was so much the issue as that they wouldn't record *any* outcomes unless the person was also withdrawing, in which case we have a separate flow for that.

Reply:
Good shout @edwardhorsford.